### PR TITLE
CML Generation

### PR DESCRIPTION
**Pull request by adam-vessey**

### CODE REVIEW

File: includes/admin.form.inc [hunk 1]
```diff
@@ -25,6 +25,7 @@ function islandora_chemistry_admin_settings_form(array $form, array &$form_state
   };
   $python = $get_default_value('islandora_chemistry_python', '/usr/bin/python');
   $obabel = $get_default_value('islandora_chemistry_openbabel', '/usr/bin/obabel');
+  $obprop = $get_default_value('islandora_chemistry_obprop', '/usr/bin/obprop');
```
Comment (willtp87) [inline includes/admin.form.inc:28]: Uninstall? -- handled

File: includes/commands/checkmol.inc [hunk 2]
```diff
@@ -159,6 +159,7 @@ class Checkmol {
     $this->send(static::HASHES);
     $found_final_hashes = FALSE;
     $output = array();
+    $count = 0;
```
Comment (willtp87) [inline includes/commands/checkmol.inc]: Not used.

Comment (adam-vessey) [inline includes/commands/checkmol.inc]: Derp, debug stuff forgotten in.

File: includes/derivatives.inc [hunk 3]
```diff
@@ -166,6 +168,38 @@ function islandora_chemistry_derive_file_from_openbabel(AbstractObject $object,
...
+ *
+ * @return string
+ *   A string, which has been populated with the results of the OpenBabel call.
+ */
+function islandora_chemistry_derive_string_from_openbabel(AbstractObject $object, $source, $input_options) {
+  module_load_include('inc', 'islandora', 'includes/utilities');
+  module_load_include('inc', 'islandora_chemistry', 'includes/commands/openbabel');
+
+  $file = islandora_chemistry_derive_download_as_temp($object[$source]);
+  $name = drupal_tempnam('temporary://', 'islandora_chemistry');
```
Comment (willtp87) [inline includes/derivatives.inc]: Not used?

**Pull request merged**